cardigan/stories/components/License.js

```diff
@@ -1,34 +1,16 @@
 import { storiesOf } from '@storybook/react';
 import License from '../../../common/views/components/License/License';
-import getLicenseInfo from '../../../common/utils/get-license-info';
+import { getLicenseInfo } from '../../../common/utils/licenses';
 import Readme from '../../../common/views/components/License/README.md';
 import { select } from '@storybook/addon-knobs';
 
 const LicenseExample = () => {
-  const licenseType = select(
+  const license = select(
     'License types',
-    [
-      'CC-0',
-      'CC-BY',
-      'CC-BY-NC',
-      'CC-BY-NC-ND',
-      'CC-BY-ND',
-      'CC-BY-SA',
-      'CC-BY-NC-SA',
-      'PDM',
-      'OGL',
-      'OPL',
-      'in-copyright',
-      'inc',
-    ],
+    ['CC-0', 'CC-BY', 'CC-BY-NC', 'CC-BY-NC-ND', 'PDM'],
```

Comment:
Why are there fewer licences here? Is it because there are the ones to which we add information (i.e `humanRedableText`)?

Reply:
No, these are just the subset of licenses that the content editors currently use and therefore we have all the data in code and so can display them in Cardigan. I could add all the licenses for the content editors, as James has suggested here: https://github.com/wellcometrust/wellcomecollection.org/pull/5076#discussion_r376986069